core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -189,7 +189,6 @@ private ColumnType() {
     }
 
     public static int geohashWithPrecision(int bits) {
-        assert bits > 0;
```

Comment:
Asserts are not executed at prod environment, there is usually no reason to remove them unless they are meaningless. Any reason to remove this one?

Reply:
It should be assert bits > 0 && bits < 61;
But there is a lot of code with this check explicit